installer/install.sh

```diff
@@ -60,6 +60,7 @@ init_helm
 # helm repos.
 helm repo add lightbend-helm-charts https://repo.lightbend.com/helm-charts/ > /dev/null 2>&1
 helm repo add strimzi http://strimzi.io/charts/ > /dev/null 2>&1
+helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator > /dev/null 2>&1
```

Comment:
Should I close https://github.com/lightbend/cloudflow/pull/8? I would rather aim for updating, moving https://github.com/lightbend/fdp-sparkoperator to https://hub.helm.sh/, as central helm charts are being deprecated (https://github.com/helm/charts#deprecation-timeline).

Reply:
Right. I do remember that the current Helm repo is being deprecated and we need to add the chart to the hub. That can be a separate task that is implemented in parallel to this PR.

Reply:
The question is whether we want to do this now as other people have done so we dont do double work.

Reply:
Agreed. We'll need @agemooij  to green light it ;)

Reply:
@skonto @agemooij I think we should start the process of migrating the chart to https://hub.helm.sh/, because as Stavros said, the deprecation deadline is approaching. In my opinion, we should simply discard https://github.com/lightbend/fdp-sparkoperator. We just put https://github.com/helm/charts/tree/master/incubator/sparkoperator to Helm hub. If there's anything useful in the former, we take it and put it in the latter as we migrate. 

The same thing should be done for our flink-operator chart. Someone already [asked](https://github.com/lightbend/flink-operator/issues/2#issuecomment-577572034) to help.

Reply:
@yuchaoran2011 if we put  https://github.com/helm/charts and then that repo is depracated how we are going to do modifications? I think we need to migrate to our own repo which means rename and keep what we have, as people did here: https://github.com/helm/charts/issues/18781

Reply:
What I meant was copy https://github.com/helm/charts/tree/master/incubator/sparkoperator, host it ourselves at Lightbend and then put it to Helm hub.

Reply:
cool